SigningService/IPackagePartSigner.cs

```diff
@@ -1,9 +1,13 @@
+using System.IO;
 using System.IO.Packaging;
+using System.Security.Cryptography;
```

Comment:
I don't think we need SHA256 in the interface, right? this is something specific to the StrongNameSigner implementation of this interface.